recipe/install_cuda.bat

```diff
@@ -169,6 +170,14 @@ set "CUDA_INSTALLER_CHECKSUM=a2388d0044b2dd6a3469938eb6108c85"
 set "CUDA_COMPONENTS=%CUDA_COMPONENTS% nsight_nvtx_%CUDA_VERSION%"
 goto cuda_common
 
+:cuda118
+set "CUDA_NETWORK_INSTALLER_URL=https://developer.download.nvidia.com/compute/cuda/11.8.0/network_installers/cuda_11.8.0_windows_network.exe"
+set "CUDA_NETWORK_INSTALLER_CHECKSUM=600ca859835a37395277a5f3a5b6037d"
+set "CUDA_INSTALLER_URL=https://developer.download.nvidia.com/compute/cuda/11.8.0/local_installers/cuda_11.8.0_522.06_windows.exe"
+set "CUDA_INSTALLER_CHECKSUM=acfd3588b31b33aa8c639c718d343c4e"
```

Comment:
In [this CI build]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=620564&view=logs&j=4eca6a2e-7d0b-56b5-2d2a-a8aca6987f82&t=9832422a-f5c0-51e9-a268-92bade59545b ) (also [attached log]( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/files/10135781/nvcc-feedstock_win_64_cuda_compiler_version11.8_20221130.2.txt )), got a checksum mismatch.

When looking at [the checksums here]( https://developer.download.nvidia.com/compute/cuda/11.8.0/docs/sidebar/md5sum.txt ), am also seeing a different value. Think the checksum listed here was for the WSL installer. Given this, think we need to change this one like so:

```suggestion
set "CUDA_INSTALLER_CHECKSUM=894c61ba173d26dc667e95ee734d3c5a"
```

While looking at the checksums, confirmed the network installer checksum was correct.

Reply:
Can do, I'll prep a follow-up patch

Reply:
Submitted PR ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/218 )